lib/msal-node/docs/caching.md

```diff
@@ -38,25 +38,25 @@ cca.acquireTokenSilent(silentTokenRequest).then((response) => {
 
 In production, you would most likely want to serialize and persist the token cache. Depending on the type of application, you can:
 
-* Desktop apps, console apps (public clients):
+* Desktop apps, console apps (public clients apps (PCA)):
   * Use [MSAL Node Extensions](../../../extensions/msal-node-extensions/README.md), which provides persistence and encryption at rest solutions on Windows, Linux and Mac OS
-* Web apps, web APIs, daemon apps (confidential client):
+* Web apps, web APIs, daemon apps (confidential client apps (CCA)):
   * MSAL's in-memory token cache does not scale for production. Use the [distributed token caching](#performance-and-security) pattern to persist the cache in your choice of storage environment (Redis, MongoDB, SQL databases etc. -keep in mind that you can use these in tandem *e.g.* a Redis-like memory cache as a first layer of persistence, and a SQL database as a second, more stable persistence layer)
 
-## Cache in memory
+## In-memory cache
 
-If you choose not to persist the cache, the `TokenCache` interface is still available to access the tokens cached in memory. The lifetime of in-memory cache is the same as MSAL instance. If the MSAL instance restarts, the cache is erased when the process lifecycle finishes. We recommend persisting the cache with encryption for all real life applications both for security and desired cache longevity.
+By default, MSAL has an in-memory cache. The in-memory cache is the single source of truth for all MSAL activities involving tokens and accounts. The lifetime of in-memory cache is the same as the MSAL instance. If the MSAL instance restarts, the cache is erased when the process lifecycle finishes. We recommend persisting the cache with encryption for all production applications both for security and desired cache longevity. If you choose not to persist the cache, the [TokenCache](https://azuread.github.io/microsoft-authentication-library-for-js/ref/classes/_azure_msal_node.tokencache.html) interface is still available to access the cached entities.
```

Comment:
```suggestion
MSAL maintains an in-memory cache. The in-memory cache is the single source of truth for all MSAL activities involving tokens and accounts. The lifetime of in-memory cache is the same as the MSAL application object. If the process using MSAL restarts, the cache is erased when the process lifecycle finishes. This affects user scenarios as users will have to re-authenticate, and although the browser will do most some of the work, the end user experience suffers. Service to service scenarios suffer because getting a token from AAD involves HTTP requests, and is much slower than getting a token from a cache.

 We recommend persisting the cache with encryption for all production applications both for security and desired cache longevity. If you choose not to persist the cache, the [TokenCache](https://azuread.github.io/microsoft-authentication-library-for-js/ref/classes/_azure_msal_node.tokencache.html) interface is still available to access the cached entities.
```